lib/sqlmap/sqlmap_manager.rb

```diff
@@ -8,43 +8,76 @@ def initialize(session)
 
     def new_task
       res = @session.get('/task/new')
-      return JSON.parse(res.body)
+      if res && res.body 
```

Comment:
@Meatballs1 Wouldn't that trigger stacktrace when calling parse(res.body) if res is null?